packages/handlersjs-core/lib/daemons/object-scheduler.ts

```diff
@@ -0,0 +1,25 @@
+import { Scheduler } from './scheduler';
+
+/**
+ * Scheduler that takes an object & method name instead of a task, created for ComponentJs compatibility
+ */
+export class ObjectScheduler<N extends string, T extends { [n in N]: () => unknown }> extends Scheduler {
```

Comment:
Okay, this looks good, except
- it will only accept objects with ONLY that kind of key-value pairs;
- I'd prefer the function return value to be void;
- I'm not sure ComponentsJS will like this any better ... Did you try that?

Reply:
If it does not work this way, just try the handler way I suggested yesterday (that's a pattern I have had in my mind for a few months now anyway).

Reply:
> I'm not sure ComponentsJS will like this any better ... Did you try that?

Yes, it builds just fine like this, seems like componentsJs doesn't care about stuff it can't read if its a generic type, only fails if it can't read types from constructor arguments

Similar to syncservice